jupyter_server_ydoc/handlers.py

```diff
@@ -28,16 +29,22 @@ class JupyterTempFileYStore(TempFileYStore):
     prefix_dir = "jupyter_ystore_"
 
 
-class JupyterSQLiteYStore(SQLiteYStore):
-    db_path = ".jupyter_ystore.db"
-    document_ttl = 24 * 60 * 60
+def sqlite_ystore_factory(
```

Comment:
If we do insist on keeping traits on the extension app class and not defining a new subclass of `SQLiteYStore` at file-scope, I would like this function to be renamed to `sqlite_ystore_class_factory`. Factories typically involve creation of objects, not classes (which are technically objects in Python too, but it's better to be as narrow as possible with our names).

Reply:
Yes, I think the name is not well chosen. Actually I don't even think we need a factory. I'll change that :+1: